backend/tests/unit/records/marshmallow/literature/common/test_publication_info_item.py

```diff
@@ -40,6 +42,7 @@ def test_returns_non_empty_fields_if_pubinfo_freetext_present():
     assert expected == json.loads(result)
 
 
+@pytest.mark.xfail(reason="It's not a correct behaviour anymore.")
```

Comment:
why not remove this?

Reply:
As I do not have any requirement or written proof that this is actually not correct.  ;)

Reply:
as discussed IRL, this is the correct behavior.